From d74fc29cb7e97c02364bf9b68ee7a807484b770f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <marcus.mueller@student.kit.edu>
Date: Fri, 31 May 2013 18:24:17 +0200
Subject: [PATCH 01/31] added utf8 encoding line to python templates

python breaks if it encounters special characters
in files without specified encoding.
Which is especially funny if you've got umlauts
in your name.
---
 gr-utils/python/modtool/templates.py |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/gr-utils/python/modtool/templates.py b/gr-utils/python/modtool/templates.py
index 3ce31cd..227a8d3 100644
--- a/gr-utils/python/modtool/templates.py
+++ b/gr-utils/python/modtool/templates.py
@@ -303,6 +303,7 @@ namespace gr {
 
 # Python block
 Templates['block_python'] = '''\#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 ${str_to_python_comment($license)}
 #
 #if $blocktype == 'noblock'
@@ -457,6 +458,7 @@ namespace gr {
 
 # Python QA code
 Templates['qa_python'] = '''\#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 ${str_to_python_comment($license)}
 #
 
-- 
1.7.10.4

